assets/js/modules/analytics-4/components/widgets/AudienceSegmentation/AudienceTile.js

```diff
@@ -0,0 +1,349 @@
+		TileIcon,
+		tileTitle,
+		topCities: topCitiesColumns,
+	} ) {
+		return (
+			<div className="googlesitekit-audience-segmentation-tile__metric googlesitekit-audience-segmentation-tile__cities-metric">
+				<div className="googlesitekit-audience-segmentation-tile__metric-icon">
+					<TileIcon />
+				</div>
+				<div className="googlesitekit-audience-segmentation-tile__metric-container">
+					<div className="googlesitekit-audience-segmentation-tile__metric-title">
+						{ tileTitle }
+					</div>
+					<div className="googlesitekit-audience-segmentation-tile__metric-content">
+						{ topCitiesColumns === null && <NoData /> }
+						{ topCitiesColumns &&
+							topCitiesColumns.dimensionValues.map(
+								( city, index ) => (
+									<div
+										key={ city.value }
+										className="googlesitekit-audience-segmentation-tile__cities-metric-city"
+									>
+										<div className="googlesitekit-audience-segmentation-tile__cities-metric-city-name">
+											{ city.value }
+										</div>
+										<div className="googlesitekit-audience-segmentation-tile__cities-metric-city-value">
+											{ numFmt(
+												topCitiesColumns.metricValues[
+													index
+												].value,
+												{
+													style: 'percent',
+													maximumFractionDigits: 1,
+												}
+											) }
+										</div>
+									</div>
+								)
+							) }
+					</div>
+				</div>
+			</div>
+		);
+	}
+
+	function ContentLinkComponent( { content } ) {
+		const contentTitle = topContentTitles[ content.value ];
+		const url = content.value;
+
+		const serviceURL = useSelect( ( select ) => {
+			return ! viewOnlyDashboard
+				? select( MODULES_ANALYTICS_4 ).getServiceReportURL(
+						'all-pages-and-screens',
+						{
+							filters: {
+								unifiedPagePathScreen: url,
+							},
+							dates,
+						}
+				  )
+				: null;
+		} );
+
+		if ( viewOnlyDashboard ) {
+			return (
+				<div className="googlesitekit-audience-segmentation-tile__top-content-metric-name">
+					{ contentTitle }
+				</div>
+			);
+		}
+		return (
+			<Link
+				href={ serviceURL }
+				title={ title }
+				external
+				hideExternalIndicator
+			>
+				{ contentTitle }
+			</Link>
+		);
+	}
+
+	function AudienceTilePagesMetric( {
+		TileIcon,
+		tileTitle,
+		topContent: topContentRows,
+	} ) {
+		return (
+			<div className="googlesitekit-audience-segmentation-tile__metric googlesitekit-audience-segmentation-tile__top-content-metric">
+				<div className="googlesitekit-audience-segmentation-tile__metric-icon">
+					<TileIcon />
+				</div>
+				<div className="googlesitekit-audience-segmentation-tile__metric-container">
+					<div className="googlesitekit-audience-segmentation-tile__metric-title">
+						{ tileTitle }
+						{ topContentRows !== null && (
+							<div className="googlesitekit-audience-segmentation-tile__top-content-partial-data-badge">
+								{ __( 'Partial data', 'google-site-kit' ) }
```

Comment:
Hmm. Although I see it's specced in the IB, it doesn't really make so much sense to implement this badge here in this manner. The "Partial data" badge is also rendered at the top of the tile when the connected property is in the partial data state, so we're going to want a reusable component for it.

We also have a separate issue for the implementation of the badges: https://github.com/google/site-kit-wp/issues/8142

I guess seeing as it has been implemented here we _could_ keep it and then refactor it to a separate component as part of #8142. Although, I would have thought we'd also want to consider reusing the `Badge` component for it.

![image](https://github.com/google/site-kit-wp/assets/18395600/1f716c40-8f1b-4e22-8f8b-8b2ed697ffd2)

Reply:
I'm happy with either approach, I think it would be cleaner to remove from this PR and work on them later in #8142

Reply:
Thanks @benbowler. Tbh, so do I. Please do go ahead and remove them from this PR, in that case.